readme.md

```diff
@@ -81,29 +92,200 @@ You can then import your `_config.scss` file wherever you need to access a funct
 
 // Import dependencies:
 // - _config.scss to override the SCSS toolkit's defaults
-// - sanitize.css to get some nice defaults
+// - @studiometa/scss-toolkit/components/reset to get some nice defaults
 // - @studiometa/scss-toolkit with the `$has-classes` variable set to `true`
 //   for the functions, mixins and classes helpers
 @import './config';
-@import '~sanitize.css';
+@import '~@studiometa/scss-toolkit/components/reset';
 $has-classes: true;
-@import '@studiometa/scss-toolkit';
+@import '~@studiometa/scss-toolkit';
 $has-classes: false;
 
 // ...
 ```
 
 > ⚠️ We reset the `$has-classes` variable to `false` right after the toolkit import to make sure future import in any SCSS file will only import the mixins, functions and variables declarations without the class helpers.
 
-## TODO
-
-- Write documentation
-- Improve configuration 
-- Improve the grid component
-  + Add some flex helpers or create a flex grid component
-- Improve the typography configuration
-  + Add keys to the font-sizes map
-  + Add font-weight to the font-sizes map
-- Rename the visibility class prefix to `display-`
-- Rename functions with a more comprehensive name and create one letter aliases
-- Add a reset component
+## Documentation
+
+### Framework
+
+The micro-framework is composed of 8 different files which defines each a set of variables, mixins, functions and classes. Find below what each file is responsible for.
+
+#### [`framework/_breakpoints.scss`](https://github.com/studiometa/scss-toolkit/blob/master/src/framework/_breakpoints.scss)
+
+**Definitions**
+
+- `$breapoints`: a map of names and values of breakpoints
+- `@function media($breakpoint, $type, $unit)`: a function to get a breakpoint declaration given a name
+- `@function md($breakpoint, $type, $unit)`: an alias for the `media(...)` function
+
+**Defaults**
+
+```scss
+$breakpoints: (
+  'xxs': 0,
+  'xs': 480,
+  's': 768,
+  'm': 1024,
+  'l': 1280,
+  'xl': 1440,
+  'xxl': 1920,
+) !default;
+```
+
+**Usage**
+
+```scss
+.foo {
+  display: none;
+
+  // Media queries
+  @media #{media('s')} { // @media (min-width: 48em) { ... }
+    display: block;
+  }
+
+  @media #{md('xs', 'max')} { // @media not all and (min-width: 48em) { ... }
```

Comment:
max-width dans le commentaire non ?

Reply:
Le paramètre `max` n'utilise pas vraiment `max`, mais une négation de `min`, d'après une idée trouvée sur Codepen qui permet d'utiliser `min` et `max` sans conflits : [https://codepen.io/vincent-valentin/pen/ObBJae](https://codepen.io/vincent-valentin/pen/ObBJae)

Reply:
The Codepen example has been added to the file for more clarity 🙂